application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadAutoArchiver.java

```diff
@@ -123,4 +120,20 @@ private static boolean shouldBeArchived(MessageChannel channel, Instant archiveA
 
         return lastActivity.isBefore(archiveAfterMoment);
     }
+
+    private void handleArchiveFlow(ThreadChannel threadChannel, MessageEmbed embed) {
+        Consumer<Throwable> handleFailure = error -> {
+            if (error instanceof ErrorResponseException) {
+                logger.warn("Unknown error occurred during help thread auto archive routine",
+                        error);
+            }
+        };
+
+        threadChannel.getGuild()
+            .retrieveMemberById(threadChannel.getOwnerIdLong())
+            .flatMap(author -> threadChannel.sendMessage(author.getAsMention()).addEmbeds(embed))
```

Comment:
when it's a transferred question, the bot will ping itself. _Note for another pr_